_gcode/M600.md

```diff
@@ -0,0 +1,80 @@
+---
+author: clexpert, petrzjunior
+category: [ 'gcode' ]
+
+
+title: Filament Change
+brief: Automatically change filament
+
+experimental: false
+since: 1.0.0-beta
```

Comment:
Oh, is it version of the documentation? I spend half an hour searching when `M600` was first implemented.